### PR TITLE
update free drive and active navigation feedback events

### PR DESCRIPTION
**Pull request by Zayankovsky**
### Description
Updated feedback events for free drive and active navigation: removed unnecessary and added missing ones. 

cc @azarovalex 

### Changelog

```
<changelog>Updated feedback events for free drive and active navigation</changelog>
```

### CODE REVIEW

**Comment by RingerJK** [comment]
`ROAD_KNOW_BY_DIFFERENT_NAME` has to be removed
https://github.com/mapbox/mapbox-navigation-android/pull/4794/files#diff-2a926aad48b2853458312661b52f68514e6de8cfa63a1821897c8b04e71426c7R123

**Comment by RingerJK** [comment]
`TURN_AT_INTERSECTION_WAS_UNPROTECTED` also must be removed

**Comment by Zayankovsky** [comment]
> `ROAD_KNOW_BY_DIFFERENT_NAME` has to be removed
> https://github.com/mapbox/mapbox-navigation-android/pull/4794/files#diff-2a926aad48b2853458312661b52f68514e6de8cfa63a1821897c8b04e71426c7R123



> `TURN_AT_INTERSECTION_WAS_UNPROTECTED` also must be removed

Removed those and `ROAD_IS_MISSING_FROM_MAP` as well.

**Review by RingerJK — APPROVED** [review]
great work 🚀

File: libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/FeedbackEvent.kt [hunk 1]
```diff
@@ -203,22 +184,36 @@ object FeedbackEvent {
...
     const val CARS_NOT_ALLOWED_ON_STREET = "cars_not_allowed_on_street"
 
-    /**
-     * Feedback description for *routing error*: turn at intersection was unprotected
-     */
-    const val TURN_AT_INTERSECTION_WAS_UNPROTECTED =
-        "turn_at_intersection_was_unprotected"
-
     /**
      * Feedback description for *routing error*: street permanently blocked off
```
Comment (azarovalex) [inline libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/FeedbackEvent.kt:188]: The category should be *road issue*?

Comment (RingerJK) [reply libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/FeedbackEvent.kt:188]: road issue exists `line#20` https://github.com/mapbox/mapbox-navigation-android/pull/4794/files#diff-2a926aad48b2853458312661b52f68514e6de8cfa63a1821897c8b04e71426c7R20

File: libnavigation-core/api/current.txt [hunk 2]
```diff
@@ -569,52 +568,52 @@ package com.mapbox.navigation.core.telemetry.events {
...
     field public static final String ROUTE_INCLUDED_MISSING_ROADS = "route_included_missing_roads";
-    field public static final String ROUTE_NOT_DRIVE_ABLE = "route_not_drive_able";
+    field public static final String ROUTE_NOT_ALLOWED = "route_not_allowed";
+    field public static final String ROUTE_NOT_DRIVE_ABLE = "route_not_driveable";
     field public static final String ROUTE_NOT_PREFERRED = "route_not_preferred";
     field public static final String ROUTING_ERROR = "routing_error";
     field public static final String STREET_NAME_INCORRECT = "street_name_incorrect";
     field public static final String STREET_PERMANENTLY_BLOCKED_OFF = "street_permanently_blocked_off";
-    field public static final String TURN_AT_INTERSECTION_WAS_UNPROTECTED = "turn_at_intersection_was_unprotected";
+    field public static final String STREET_TEMPORARY_BLOCKED_OFF = "street_temporary_blocked_off";
```
Comment (azarovalex) [inline libnavigation-core/api/current.txt:600]: "temporarily" should be used instead of "temporary", lets change both the constant name and value

Comment (RingerJK) [reply libnavigation-core/api/current.txt:600]: it was already used with 1.x and as far as I see iOS uses the same conts https://github.com/mapbox/mapbox-navigation-ios/blob/main/Sources/MapboxCoreNavigation/PassiveNavigationFeedbackType.swift#L53

Comment (RingerJK) [reply libnavigation-core/api/current.txt:600]: why would you change that?

Comment (azarovalex) [reply libnavigation-core/api/current.txt:600]: On iOS it was merged yesterday, but not we've caught this typo and there is a quick fix PR before the release: https://github.com/mapbox/mapbox-navigation-ios/pull/3337
I don't get how this constant was already used in 1.x if this PR is one day old.🤔

File: libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/FeedbackEvent.kt (hunk 1 above)
Comment (azarovalex) [reply libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/FeedbackEvent.kt:188]: I mean in the comment. For `const val STREET_PERMANENTLY_BLOCKED_OFF` the comment is `routing error` and for `const val STREET_TEMPORARY_BLOCKED_OFF` it's `road issue`.

File: libnavigation-core/api/current.txt (hunk 2 above)
Comment (RingerJK) [reply libnavigation-core/api/current.txt:600]: right, mixed up it with `*permanently*`. @Zayankovsky could you address the issue with this for Android?

Comment (Zayankovsky) [reply libnavigation-core/api/current.txt:600]: Sure thing.

**Pull request merged**